local/src/test/scala/com/salesforce/op/local/OpWorkflowModelLocalTest.scala

```diff
@@ -57,14 +56,31 @@ import org.slf4j.LoggerFactory
 
 
 @RunWith(classOf[JUnitRunner])
-class OpWorkflowModelLocalTest extends FlatSpec with PassengerSparkFixtureTest with TestCommon {
-
+class OpWorkflowModelLocalTest extends FlatSpec with TestSparkContext with TempDirectoryTest with TestCommon {
```

Comment:
Where do you use temp directory here?

Reply:
For example, here: https://github.com/salesforce/TransmogrifAI/blob/30cd461968ab9df9648afc7e6eb7f31a9d8f155f/local/src/test/scala/com/salesforce/op/local/OpWorkflowModelLocalTest.scala#L207

The test originally pulled it in via `PassengerSparkFixtureTest`, but I only needed things in `TempDirectoryTest`.

Reply:
Got it.